helpers.go

```diff
@@ -6,6 +6,26 @@ import (
 	"github.com/medama-io/go-useragent/internal"
 )
 
+// GetDevice returns the device type as a string.
+func (ua UserAgent) GetDevice() string {
+	switch ua.device {
+	case internal.DeviceDesktop:
+		return internal.DeviceDesktopStr
```

Comment:
suggestion: you should be able to re-use the consts declared here instead.

 https://github.com/medama-io/go-useragent/blob/16197f4a11d97d08061898717f12d19dcf6d22c6/internal/match.go#L36